### ISSUE TITLE
Don't cache certain elements on page (like alert & notice)

### ISSUE THREAD

**Issue by marczking** [rails/turbolinks#560]
When I sign in to my page I get a notice "Successfully Signed In". When I then navigate to an other sub-page of my site and navigate back with my `back-button` my notice is shown again.

How can I prevent this? Basically setting some element in the page I want not to be cached.

**Comment by naveed-ahmad** [comment]
Add `data-turbolinks-temporary` to DOMs you don't want to cache, not tested though :) Playing with turbolinks now a days.

**Comment by marczking** [comment]
Thanks for your suggestion. Unfortunately I am not able to **not** cache certain parts of my DOM.

I tried `data-turbolinks-temporary` and `data-no-turbolink`. Both without success.

While using partial replacement with JS according to the docs I can:
```
Turbolinks.visit(url, { cacheRequest: false });
```

My idea is you could add a `data-cache-request=false`. Or one of the above to work. For now I'll just live with notices reappearing every once in a while...

**Comment by naveed-ahmad** [comment]
Ok, maybe its just browser cache. Did you closed the alert before navigating to other page?

**Comment by marczking** [comment]
Yeah, I did. It just caches the element in the current state. I'm not sure if it is possible to cache the whole page except some element.

For that one would need to remove the element *first* and then cache it. Maybe worth considering in a future version of turbolinks :smiley:

**Comment by mosinski** [comment]
:+1: same in my case I trying to deal with cached notices.
Maybe there's now any solution for this problem :question:

**Comment by nuflowx** [comment]
I haven't had a look on the new turbolinks, but maybe it's handled differently with that one.?

**Comment by mosinski** [comment]
**Gem version**:
```Using turbolinks 5.0.0.beta2```

I'm testing now [Preparing the Page to be Cached](https://github.com/turbolinks/turbolinks#preparing-the-page-to-be-cached) perhaps this is the solution.

**Comment by nitinkumar24** [comment]
I am facing the similar issue.
 error and notice are showing Multiple times
@marczking did you find any solution?
